modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache

```diff
@@ -39,8 +39,8 @@
         <DocumentationFile>bin\Release\{{packageName}}.XML</DocumentationFile>
     </PropertyGroup>
     <ItemGroup>
-        <Reference Include="Nancy, Version=1.4.1.0, Culture=neutral, processorArchitecture=MSIL">
-            <HintPath>..\..\packages\Nancy.1.4.1\lib\net40\Nancy.dll</HintPath>
+        <Reference Include="Nancy, Version=1.4.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I know it's weird, but I actually did a nuget update-package on the petstore example and this is exactly how it modified the project file. It compiled so i didn't ask any questions.